src/components/navBar/NavLink.tsx

```diff
@@ -0,0 +1,30 @@
+type NavLinkProps = {
+  href: string
+  name: string
+  icon: JSX.Element
+  currentPage: boolean
+  setCurrentPage: (name: string) => unknown
+}
+
+export default function NavLink({
+  href,
+  name,
+  icon,
+  currentPage,
+  setCurrentPage
+}: NavLinkProps) {
+  const tabClasses =
+    'inline-block w-full justify-center pt-2 pb-1 text-center hover:text-poops-red'
+  const currentTabClasses = tabClasses + ' border-t-2 border-t-poops-red'
+
+  return (
+    <a
```

Comment:
Should be using Next link as it is internal routing, the only time you wouldn't want to be using it is when opening a new tab.

Reply:
I've updated it so that it uses Next Link, let me know if you have any more problems :)